auth-web/src/util/common-util.ts

```diff
@@ -1,9 +1,35 @@
 /**
  * Place to put all the custom utility methods
  */
-const URL_MATCHER = new RegExp('^(?:\\w+:)?\\/\\/([^\\s\\.]+\\.\\S{2}|localhost[\\:?\\d]*)\\S*$')
-export default {
-  isUrl (value:string):boolean {
+export default class CommonUtils {
+  // checking url matches the regex
+  static isUrl (value:string):boolean {
+    const URL_MATCHER = new RegExp('^(?:\\w+:)?\\/\\/([^\\s\\.]+\\.\\S{2}|localhost[\\:?\\d]*)\\S*$')
     return URL_MATCHER.test(value)
   }
+  // formatting incorporation number according to the length of numbers
+  static formatIncorporationNumber (incorpNum:string, numLength?:number):string {
+    numLength = numLength || 7 // optional: go with '7' if nothing specified
```

Comment:
do you wanna consider writing some tests for this? It will help in future when we have to make changes.. just invoke this function with valid and invalid inputs and do an assertion... notning complicated